perllib/FixMyStreet/DB/Result/ResponsePriority.pm

```diff
@@ -28,6 +28,8 @@ __PACKAGE__->add_columns(
   { data_type => "text", is_nullable => 1 },
   "external_id",
   { data_type => "text", is_nullable => 1 },
+  "is_default",
+  { data_type => "boolean", default_value => \"false", is_nullable => 0 },
```

Comment:
i'm guessing as the MD5 hash below didn't change, this wasn't generated by running `db/rerun_dbic_loader.pl` :) Which doesn't really matter, as it seems like column order in different dev databases causes changes there anyway, but if running it adds this and doesn't change anything else but update the hash, might be worth doing.